src/app/rundown-view/components/minishelf/mini-shelf.component.html

```diff
@@ -0,0 +1,27 @@
+<div class="c-mini-shelf">
+  <div
+    class="c-mini-shelf__part"
+    *ngFor="let part of segment?.parts;"
```

Comment:
From the class definition, the segment input variable is required, so the optional chaining her should be removed.